server/repository/src/db_diesel/assets/asset_row.rs

```diff
@@ -0,0 +1,100 @@
+use super::asset_row::asset::dsl::*;
+
+use crate::RepositoryError;
+use crate::StorageConnection;
+
+use chrono::NaiveDate;
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+
+table! {
+    asset (id) {
+        id -> Text,
+        store_id -> Text,
+        serial_number -> Text,
+        asset_category_id -> Text,
+        asset_type_id -> Text,
+        catalogue_item_id -> Nullable<Text>,
+        installation_date -> Nullable<Date>,
+        replacement_date -> Nullable<Date>,
+        created_datetime -> Timestamp,
+        modified_datetime -> Timestamp,
+        deleted_datetime -> Nullable<Timestamp>,
+    }
+}
+
+#[derive(Clone, Insertable, Queryable, Debug, PartialEq, AsChangeset, Eq, Default)]
+#[table_name = "asset"]
+pub struct AssetRow {
+    pub id: String,
+    pub store_id: String,
+    pub serial_number: String,
+    #[column_name = "asset_category_id"]
```

Comment:
Question: Is it good to get the database as `asset_category_id` I like that it's not then confused with some other category_id in the db level, but also makes things slightly clunky like this...